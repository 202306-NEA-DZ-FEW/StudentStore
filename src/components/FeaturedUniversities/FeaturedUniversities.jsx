import React from "react";
import Image from "next/image";

const universities = [
    { id: 1, name: "arkansas", logo: "/arkansas.svg" },
    { id: 2, name: "CIT", logo: "/CaliforniaIT.svg" },
    { id: 3, name: "Duke", logo: "/duke.svg" },
    { id: 4, name: "glasgow", logo: "/glasgow.svg" },
    { id: 5, name: "MIT", logo: "/mit.svg" },
    { id: 6, name: "Nottingham", logo: "/Nottingham.svg" },
    { id: 7, name: "Oxford", logo: "/oxford.svg" },
    { id: 8, name: "Princeton", logo: "/princeton.svg" },
    { id: 9, name: "Purdue", logo: "/purdue.svg" },
    { id: 10, name: "Stanford", logo: "/stanford.svg" },
    { id: 11, name: "Columbia", logo: "/columbia.svg" },
    { id: 12, name: "Chicago", logo: "/chicago.svg" },
    { id: 13, name: "Brown", logo: "/brown.svg" },
    { id: 14, name: "glasgow", logo: "/glasgow1.svg" },
    { id: 15, name: "liverpool", logo: "/liverpool.svg" },
    { id: 16, name: "Pennsylvania", logo: "/Pennsylvania.svg" },
    { id: 17, name: "UIA", logo: "/UIA.svg" },
];

function FeaturedUniversities() {
    return (
        <div className='bg-[#F1F6FA] py-12'>
            <div className='container mx-auto university-logos-container flex flex-wrap justify-center items-center'>
                {universities.map((university) => (
                    <div
                        key={university.id}
                        className=' bg-transparentp-4 rounded-full shadow-lg hover:shadow-xl transition duration-300 transform hover:scale-105 mx-2 my-2 animate-moveLogo'
                    >
                        <img
                            src={university.logo}
                            alt={`${university.name} Logo`}
                            width={90}
                            height={90}
                            className='object-contain rounded-full'
                        />
                    </div>
                ))}
            </div>
        </div>
    );
}

export default FeaturedUniversities;
