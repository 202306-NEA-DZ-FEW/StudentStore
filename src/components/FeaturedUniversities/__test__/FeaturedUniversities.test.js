import renderer from "react-test-renderer";
import FeaturedUniversities from "../FeaturedUniversities";

it("renders correctly", () => {
    const tree = renderer.create(<FeaturedUniversities />).toJSON();
    expect(tree).toMatchSnapshot();
});
