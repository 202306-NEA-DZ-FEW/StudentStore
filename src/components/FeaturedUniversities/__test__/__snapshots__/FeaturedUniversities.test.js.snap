// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="bg-[#F1F6FA] py-12"
>
  <div
    className="container mx-auto university-logos-container flex flex-wrap justify-center items-center"
  >
    <div
      className=" bg-transparentp-4 rounded-full shadow-lg hover:shadow-xl transition duration-300 transform hover:scale-105 mx-2 my-2 animate-moveLogo"
    >
      <img
        alt="arkansas Logo"
        className="object-contain rounded-full"
        height={90}
        src="/arkansas.svg"
        width={90}
      />
    </div>
    <div
      className=" bg-transparentp-4 rounded-full shadow-lg hover:shadow-xl transition duration-300 transform hover:scale-105 mx-2 my-2 animate-moveLogo"
    >
      <img
        alt="CIT Logo"
        className="object-contain rounded-full"
        height={90}
        src="/CaliforniaIT.svg"
        width={90}
      />
    </div>
    <div
      className=" bg-transparentp-4 rounded-full shadow-lg hover:shadow-xl transition duration-300 transform hover:scale-105 mx-2 my-2 animate-moveLogo"
    >
      <img
        alt="Duke Logo"
        className="object-contain rounded-full"
        height={90}
        src="/duke.svg"
        width={90}
      />
    </div>
    <div
      className=" bg-transparentp-4 rounded-full shadow-lg hover:shadow-xl transition duration-300 transform hover:scale-105 mx-2 my-2 animate-moveLogo"
    >
      <img
        alt="glasgow Logo"
        className="object-contain rounded-full"
        height={90}
        src="/glasgow.svg"
        width={90}
      />
    </div>
    <div
      className=" bg-transparentp-4 rounded-full shadow-lg hover:shadow-xl transition duration-300 transform hover:scale-105 mx-2 my-2 animate-moveLogo"
    >
      <img
        alt="MIT Logo"
        className="object-contain rounded-full"
        height={90}
        src="/mit.svg"
        width={90}
      />
    </div>
    <div
      className=" bg-transparentp-4 rounded-full shadow-lg hover:shadow-xl transition duration-300 transform hover:scale-105 mx-2 my-2 animate-moveLogo"
    >
      <img
        alt="Nottingham Logo"
        className="object-contain rounded-full"
        height={90}
        src="/Nottingham.svg"
        width={90}
      />
    </div>
    <div
      className=" bg-transparentp-4 rounded-full shadow-lg hover:shadow-xl transition duration-300 transform hover:scale-105 mx-2 my-2 animate-moveLogo"
    >
      <img
        alt="Oxford Logo"
        className="object-contain rounded-full"
        height={90}
        src="/oxford.svg"
        width={90}
      />
    </div>
    <div
      className=" bg-transparentp-4 rounded-full shadow-lg hover:shadow-xl transition duration-300 transform hover:scale-105 mx-2 my-2 animate-moveLogo"
    >
      <img
        alt="Princeton Logo"
        className="object-contain rounded-full"
        height={90}
        src="/princeton.svg"
        width={90}
      />
    </div>
    <div
      className=" bg-transparentp-4 rounded-full shadow-lg hover:shadow-xl transition duration-300 transform hover:scale-105 mx-2 my-2 animate-moveLogo"
    >
      <img
        alt="Purdue Logo"
        className="object-contain rounded-full"
        height={90}
        src="/purdue.svg"
        width={90}
      />
    </div>
    <div
      className=" bg-transparentp-4 rounded-full shadow-lg hover:shadow-xl transition duration-300 transform hover:scale-105 mx-2 my-2 animate-moveLogo"
    >
      <img
        alt="Stanford Logo"
        className="object-contain rounded-full"
        height={90}
        src="/stanford.svg"
        width={90}
      />
    </div>
    <div
      className=" bg-transparentp-4 rounded-full shadow-lg hover:shadow-xl transition duration-300 transform hover:scale-105 mx-2 my-2 animate-moveLogo"
    >
      <img
        alt="Columbia Logo"
        className="object-contain rounded-full"
        height={90}
        src="/columbia.svg"
        width={90}
      />
    </div>
    <div
      className=" bg-transparentp-4 rounded-full shadow-lg hover:shadow-xl transition duration-300 transform hover:scale-105 mx-2 my-2 animate-moveLogo"
    >
      <img
        alt="Chicago Logo"
        className="object-contain rounded-full"
        height={90}
        src="/chicago.svg"
        width={90}
      />
    </div>
    <div
      className=" bg-transparentp-4 rounded-full shadow-lg hover:shadow-xl transition duration-300 transform hover:scale-105 mx-2 my-2 animate-moveLogo"
    >
      <img
        alt="Brown Logo"
        className="object-contain rounded-full"
        height={90}
        src="/brown.svg"
        width={90}
      />
    </div>
    <div
      className=" bg-transparentp-4 rounded-full shadow-lg hover:shadow-xl transition duration-300 transform hover:scale-105 mx-2 my-2 animate-moveLogo"
    >
      <img
        alt="glasgow Logo"
        className="object-contain rounded-full"
        height={90}
        src="/glasgow1.svg"
        width={90}
      />
    </div>
    <div
      className=" bg-transparentp-4 rounded-full shadow-lg hover:shadow-xl transition duration-300 transform hover:scale-105 mx-2 my-2 animate-moveLogo"
    >
      <img
        alt="liverpool Logo"
        className="object-contain rounded-full"
        height={90}
        src="/liverpool.svg"
        width={90}
      />
    </div>
    <div
      className=" bg-transparentp-4 rounded-full shadow-lg hover:shadow-xl transition duration-300 transform hover:scale-105 mx-2 my-2 animate-moveLogo"
    >
      <img
        alt="Pennsylvania Logo"
        className="object-contain rounded-full"
        height={90}
        src="/Pennsylvania.svg"
        width={90}
      />
    </div>
    <div
      className=" bg-transparentp-4 rounded-full shadow-lg hover:shadow-xl transition duration-300 transform hover:scale-105 mx-2 my-2 animate-moveLogo"
    >
      <img
        alt="UIA Logo"
        className="object-contain rounded-full"
        height={90}
        src="/UIA.svg"
        width={90}
      />
    </div>
  </div>
</div>
`;
